bigtable/integration_test.go

```diff
@@ -2190,3 +2190,9 @@ func clearTimestamps(r Row) {
 		}
 	}
 }
+
+func deleteTable(ctx context.Context, t *testing.T, ac *AdminClient, name string) {
+	if err := ac.DeleteTable(ctx, name); err != nil {
+		t.Logf("DeleteTable: %v", err)
+	}
+}
```

Comment:
Actually looking at the bug, I think it might also make sense to retry the delete if it fails, up to a deadline. @kolea2 what do other languages do here?

Reply:
We prefix the table name with a timestamp and run a cleanup function before running any new tests to delete tables older than I think 2 hrs?

Reply:
See https://github.com/googleapis/java-bigtable/blob/master/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/AbstractTestEnv.java#L102-L113 for an example